Referentiedata.csv

```diff
@@ -989,6 +989,33 @@ INSPECTIERAPPORTSTATUS;DEF;Definitief;;;;;Onderhoud
 INSPECTIERAPPORTSTATUS;GET;Getekend;;;;;Onderhoud
 INSPECTIERAPPORTSTATUS;REV;Ter review;;;;;Onderhoud
 INSPECTIERAPPORTSTATUS;VAL;Vervallen;;;;;Onderhoud
+INSTALLATIESOORT;IAF;Inbouw afzuiginstallatie;Bij een afzuiginstallatie gaat het om een luchtafvoer met afzuiging naar buiten de woning of op basis van recirculatie met actieve koolstof- en vetfilters. Een afzuiginstallatie kan zowel een afzuig- of recirculatiekap boven de kookinstallatie zijn, als een in het aanrecht geïntegreerd afzuigsysteem.;26-07-2024;;;Vastgoed
+INSTALLATIESOORT;IKI;Inbouw kookplaat inductie;Een inbouw inductiekookplaat is een keukenapparaat dat in het werkblad wordt geïntegreerd. Het gebruikt magnetische velden om pannen direct te verwarmen, wat resulteert in snelle opwarming, nauwkeurige temperatuurcontrole, en verhoogde energie-efficiëntie in vergelijking met traditionele kookplaten.;26-07-2024;;;Vastgoed
```

Comment:
Laatste checkvraag: We gaan - in overweging van het overleg van vandaag - zowel installatiesoort als bouwkundigeruimten ondersteunen bij de klasse ruimten? Daarbij accepteren we (momenteel) dat overlap bestaat in gebruik van 'installatie' en sommige gevallen zelfs complete dubbeling in zit (i.e. "Wastafel"). ?

Reply:
Ja dat is voor nu de situatie